.github/workflows/deploy_onboarding_ms.yml

```diff
@@ -21,64 +21,48 @@ on:
           - uat
           - prod
 
-permissions:
-  packages: write
-  contents: write
-  issues: write
-  id-token: write
-  deployments: write
+env:
+  DIR: "./.container_app"
+  ENV_NAME: "${{ inputs.environment != null && inputs.environment || (github.base_ref == 'main' && 'prod' || (github.base_ref == 'develop' && 'uat' || 'dev')) }}"
```

Comment:
can you add a comment about this condition?

Reply:
done